src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,18 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+/**
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {
+
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+     throws IOException, ServletException {
+
+   response.getWriter().println("<h1>RegisterServlet GET request.</h1>");

```

Comment:
It looks like the class doesn't do anything yet right? Want to add a comment that calls this out?

Reply:
Yes, it just prints RegisterServlet GET request.  I thought that's what they wanted for now.
Here is the ss of what it does for now.
![image](https://user-images.githubusercontent.com/30271091/37496021-6d582acc-286d-11e8-9daf-0435fc34b174.png)

Reply:
Correct I was just saying you can add a comment under

Servlet class responsible for user registration.

that says something like

Note: User registration is not yet implemented. For now, the servlet just prints to the browser.